service/confirmation-repeater/src/index.ts

```diff
@@ -24,107 +29,57 @@ const job = schedule.scheduleJob('* * * * *', async () => {
     console.log("Job:", key, value);
     const actionId = key.split("-")[1];
 
-
-   if (value.attempts >= +maxRetries) {
-    await db.put('done-' + actionId, { done: false }, function (error: any) {
-      if (error) {
-        console.log("Put done:", error);
-      }
-    })
-    await db.del('action-' + actionId, function (error: any) {
-      if (error) {
-        console.log("Del action in job:", error);
-      }
-    })
-    await db.del('retry-' + actionId, function (error: any) {
-      if (error) {
-        console.log("DEl retry in error:", error);
-      }
-    })
+   if (value.attempts > maxRetries) { // attempts counts also 1st normal confirm
+     await db.put('done-' + actionId, { done: false });
```

Comment:
@1v4n4 you mixed promise style and callback style api. level supports both, promises are way better. they return value or THROW the error. Please learn more: https://dev.to/jsmanifest/callbacks-vs-promises-in-javascript-2d5k